Dockerfile.controller

```diff
@@ -25,6 +25,6 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager
 FROM gcr.io/distroless/static:nonroot
 WORKDIR /
 COPY --from=builder /workspace/manager .
-USER nonroot:nonroot
+USER 65532:65532
```

Comment:
Sorry for the long answer. Some security k8s tools like kyverno may block pods with non-numeric user like nonroot because it can`t validate uid. Nonroot user can has uid 0 inside container.
For example Dockerfile is generated by operator-sdk has numeric user.
https://sdk.operatorframework.io/docs/best-practices/pod-security-standards/#after-following-the-recommendations-to-be-restricted-my-workload-is-not-running-createcontainerconfigerror-what-should-i-do

In our case we use kyverno, so we aren`t able to use k6-operator without image rebuild.

Reply:
@yorugac  Any update on it?

Reply:
@dkhachyan, better late than never and thanks for the clarification! Could you please add then the comment on the origin of this specific UID? E.g. "as defined by distroless for nonroot" or smth.

Reply:
done